pkg/epinio/models/applications.js

```diff
@@ -429,10 +430,16 @@ export default class EpinioApplicationModel extends EpinioNamespacedResource {
       data: {
         instances:      this.configuration.instances,
         configurations: this.configuration.configurations,
+        settings:       this.configuration.settings ?? null,
         environment:    this.configuration.environment,
         routes:         this.configuration.routes,
       }
     });
+
+    // Restart app after applying new custom-values
+    if (this.configuration.settings) {
```

Comment:
Do we need to restart on create, or only edit? On edit we should only restart if the values have changed (or just change the message to recommend a restart to apply changes). Whatever we do for settings we should also do for application env vars

Reply:
i've changd this to just a warning instead. in the future we should add it as an option when saving